### PR TITLE
✨ New method for checking if a number is odd and new method for shallow copy an object

### PR DESCRIPTION
**Pull request by Namysh**

### CODE REVIEW

File: collections/object/shallow-copy-an-object.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
 ~~~ javascript
 const shallowCopy = obj => Object.assign({}, obj);
+
+// or
+const shallowCopy = obj => [...obj];
```
Comment (phuocng) [inline collections/object/shallow-copy-an-object.md]: I think it should be `{...obj}`?

Comment (Namysh) [reply collections/object/shallow-copy-an-object.md]: Yes my bad, I update the case ! By the way the same method works too in order to clone an array

**Review by Namysh — COMMENTED** [review]
Here the fix is

**Comment by phuocng** [comment]
Great. Thank you a lot, @Namysh 👍

**Pull request merged**